index.ts

```diff
@@ -54,10 +63,170 @@ const config: TSESLint.Linter.Config = {
       },
     ],
     curly: ["error"],
+    eqeqeq: ["error", "smart"],
+    "no-plusplus": ["error", { allowForLoopAfterthoughts: true }],
+    "no-alert": "error",
+    "object-shorthand": "warn",
+    "no-implicit-coercion": "warn",
```

Comment:
I'd even consider bumping this to error

Reply:
Yeah. I wonder if we want to take it a step further and disallow _explicit_ coercion too. This rule covers `!!`, but not `Boolean()`

Reply:
Upped this to an `error`. It doesn't look like such a rule exists to ban explicit forms of coercion, which may indeed be a little extreme/impractical in some cases.